ui/src/components/EntityModal/EntityModal.test.tsx

```diff
@@ -27,6 +27,18 @@ import { StandardPages } from '../../types/components/shareableTypes';
 import { server } from '../../mocks/server';
 import { invariant } from '../../util/invariant';
 
+const isTextDisabled = (field: HTMLElement | null | Element) => {
```

Comment:
Looks like custom matchers
https://github.com/epicweb-dev/epic-stack/blob/95982eeb8caec5d7477a9edd13547eac1d34c2ef/tests/setup/custom-matchers.ts

Reply:
nice idea, added it